<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1368</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Serif</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>GENEALOGY</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ressources_png/genealogy.png</normaloff>ressources_png/genealogy.png</iconset>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::ForceTabbedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1368</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu00">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menu01">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="switch01"/>
     <addaction name="switch02"/>
    </widget>
    <widget class="QMenu" name="menu02">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="switch03"/>
     <addaction name="switch04"/>
     <addaction name="separator"/>
     <addaction name="switch05"/>
     <addaction name="switch06"/>
    </widget>
    <addaction name="switch00"/>
    <addaction name="separator"/>
    <addaction name="menu01"/>
    <addaction name="separator"/>
    <addaction name="menu02"/>
    <addaction name="separator"/>
    <addaction name="switch07"/>
   </widget>
   <widget class="QMenu" name="menu04">
    <property name="title">
     <string>Genealogy</string>
    </property>
    <addaction name="switch09"/>
    <addaction name="switch10"/>
   </widget>
   <widget class="QMenu" name="menu05">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="switch11"/>
    <addaction name="switch12"/>
    <addaction name="switch13"/>
   </widget>
   <widget class="QMenu" name="menu03">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="switch08"/>
   </widget>
   <addaction name="menu00"/>
   <addaction name="menu03"/>
   <addaction name="menu04"/>
   <addaction name="menu05"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="switch07"/>
   <addaction name="separator"/>
   <addaction name="switch09"/>
   <addaction name="switch10"/>
  </widget>
  <widget class="QDockWidget" name="IndividualsList">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>600</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>200</width>
     <height>620</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDockWidget {color:blue}</string>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Individuals List</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_7">
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>571</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="IndivSelect">
        <property name="font">
         <font>
          <family>Serif</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {background:yellow; color:blue}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="IndivList">
        <property name="font">
         <font>
          <family>Courier 10 Pitch</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="Tree">
   <property name="minimumSize">
    <size>
     <width>1050</width>
     <height>330</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>1050</width>
     <height>310</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QDockWidget {color:blue; background: white}</string>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Tree</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_8">
    <property name="font">
     <font>
      <family>Serif</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <widget class="QListWidget" name="PaternelGrandFather">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>220</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="PaternelGrandMother">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>0</y>
       <width>220</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="MaternelGrandFather">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>0</y>
       <width>220</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="MaternelGrandMother">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>0</y>
       <width>220</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="Father">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>65</y>
       <width>240</width>
       <height>60</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="Mother">
     <property name="geometry">
      <rect>
       <x>575</x>
       <y>65</y>
       <width>240</width>
       <height>60</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="Individual">
     <property name="geometry">
      <rect>
       <x>335</x>
       <y>135</y>
       <width>260</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="Children">
     <property name="geometry">
      <rect>
       <x>335</x>
       <y>230</y>
       <width>260</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="BrothersandSisters">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>260</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="Spouse">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>135</y>
       <width>260</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label00">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Father</string>
     </property>
    </widget>
    <widget class="QLabel" name="label01">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>90</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Mother</string>
     </property>
    </widget>
    <widget class="QLabel" name="label02">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Brothers and sisters</string>
     </property>
    </widget>
    <widget class="QLabel" name="label03">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>110</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Spouses</string>
     </property>
    </widget>
    <widget class="QLabel" name="label04">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>250</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Children</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="Card">
   <property name="minimumSize">
    <size>
     <width>1050</width>
     <height>250</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>1050</width>
     <height>250</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QDockWidget {color:blue}</string>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Card</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QLineEdit" name="CId">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="CName">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="CFirstname">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="CSexe">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>10</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="IndividualTable">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>701</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>8</number>
     </property>
     <column>
      <property name="text">
       <string>Event</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>City</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Note</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Source</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Information</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>age</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Id</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="CPhoto">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>96</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {background:white; color:blue}</string>
     </property>
     <property name="text">
      <string>PHOTO</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="CAge">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>10</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="switch00">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="statusTip">
    <string>Open</string>
   </property>
  </action>
  <action name="switch07">
   <property name="icon">
    <iconset>
     <normaloff>ressources_png/quitter.png</normaloff>ressources_png/quitter.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="switch01">
   <property name="text">
    <string>CSV</string>
   </property>
  </action>
  <action name="switch02">
   <property name="text">
    <string>GEDCOM</string>
   </property>
  </action>
  <action name="switch03">
   <property name="text">
    <string>CSV</string>
   </property>
  </action>
  <action name="switch04">
   <property name="text">
    <string>GEDCOM</string>
   </property>
  </action>
  <action name="switch05">
   <property name="text">
    <string>HTML</string>
   </property>
  </action>
  <action name="switch06">
   <property name="text">
    <string>PDF</string>
   </property>
  </action>
  <action name="switch09">
   <property name="icon">
    <iconset>
     <normaloff>ressources_png/new.png</normaloff>ressources_png/new.png</iconset>
   </property>
   <property name="text">
    <string>New Individual</string>
   </property>
  </action>
  <action name="switch10">
   <property name="icon">
    <iconset>
     <normaloff>ressources_png/lien.png</normaloff>ressources_png/lien.png</iconset>
   </property>
   <property name="text">
    <string>New Link</string>
   </property>
  </action>
  <action name="switch11">
   <property name="text">
    <string>DbtoTxt</string>
   </property>
  </action>
  <action name="switch12">
   <property name="text">
    <string>Analyse</string>
   </property>
  </action>
  <action name="switch13">
   <property name="text">
    <string>Private/Public</string>
   </property>
  </action>
  <action name="switch08">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
